akka-stream-tests/src/test/java/akka/stream/javadsl/FlowGraphTest.java

```diff
@@ -156,6 +149,92 @@ public void apply(Integer param) throws Exception {
     assertEquals(expected2, output2);
   }
 
+  private static <T> Sink<T, Future<BoxedUnit>> createSink(final JavaTestKit probe){
+    return Sink.foreach(new Procedure<T>() {
+      @Override
+      public void apply(T param) throws Exception {
+        probe.getRef().tell(param, ActorRef.noSender());
+      }
+    });
+  }
```

Comment:
thanks, will still have private method to have this line, primarily because of "magic" second parameter.